modules/ROOT/pages/configure-client-management-openid-task.adoc

```diff
@@ -109,6 +109,8 @@ If the *Issuer* field isn't configured or the provider's discovery endpoint is u
 - Authorization
 - Refresh Token
 
+When you add a new client provider, metadata from `$ISSUER/.well-known/oauth-authorization-server` and `$ISSUER/.well-known/openid-configuration` is discovered and validated prior to merging grant types. If you want to fetch metadata for validation for an existing client provider, save that client provider again, and the validation occurs automatically.
```

Comment:
and what do you think of doing something like 
`... and the list of supported grant types is updated automatically.`
instead of `... and the validation occurs automatically.`